fvm/src/gas/mod.rs

```diff
@@ -58,6 +104,24 @@ impl GasTracker {
     }
 }
 
+/// Converts the specified gas into equivalent fractional gas units
+#[inline]
+pub fn gas_to_frgas(gas: i64) -> i64 {
+    gas * (1 << FRGAS_PRECISION)
+}
+
+/// Converts the specified fractional gas units into gas units
+#[inline]
+pub fn frgas_to_gas(frgas: i64, round_up: bool) -> i64 {
+    let p = 1 << FRGAS_PRECISION;
+
+    let mut div_result = frgas / p;
+    if round_up && frgas % p != 0 {
+        div_result = div_result.saturating_add(1);
+    }
+    div_result
+}
```

Comment:
Instead of providing conversion methods, I'd consider adding methods to the gas tracker:

- `charge_milligas`, `available_milligas`, etc.